### ISSUE TITLE
Main Display Component 

### ISSUE THREAD

**Issue by Edd-wordd** [Edd-wordd/monaTech#7]
- Need to figure out what the two btns on the main display link to 
- Associate the correct icons to them

**Comment by Edd-wordd** [comment]
Same as issue #44. Home page main display component can link to two services that the company offers. The about us page, main display component can link to FAQ and projects services section. The services page, main display component can lead to contact us form and get a quote section.

**Comment by Edd-wordd** [comment]
-For the home page I linked the first main display button to web development services and the second to SEO services.
-For the about us page I linked the first main display button to FAQ which there is no page so a page needs to be added in order for that button to work correctly. The second button was linked to "our clients' page but there is also no "our clients" page and needs to be added as well.
-For the services page I linked the first button on the main display component to contact us page and the second button was linked to get a quote but we need to get a modal going in order to get that button going as well.
I have went ahead and created the issues for those pages to get made up.
